README.md

```diff
@@ -46,6 +48,7 @@ async function main() {
   );
   // In local environment it will retrieve NS2_SENDGRID_KEY from process.env variables
   // Inside a WebApp it will retrieve SENDGRID-KEY from the NS1 keyvault
-  const emailClient = new EmailClient(await getSecret('NS2', 'SENDGRID_KEY'));
+  // The third parameter will explicitly tell if we want to use the cache or not on this particular call.
+  const emailClient = new EmailClient(await getSecret('NS2', 'SENDGRID_KEY', false));

```

Comment:
Do you want to make that an `options` object?

Reply:
No, I don't xD

I was kinda hoping that I don't have to stick more parameters in there, and since the third is optional it should be fine, because the usual case would be only the two parameters. I can change it if you want, though :P

Reply:
Yeah, I know, just thinking it'll be easier to add params later if needed.
Pretty sure it'll happen 😂

Reply:
I hope not, but if I see a fourth parameter I guess we can refactor xd.